src/idea.js

```diff
@@ -0,0 +1,19 @@
+class Idea {
+  constructor(title, body){
+    this.id = Date.now();
+    this.title = title;
+    this.body = body;
+    this.star = false;
+  }
+  saveToStorage(){
```

Comment:
Thank you for adding the shells of the necessary methods that we will be using later on.